eng/packages/http-client-csharp/ci.yml

```diff
@@ -62,4 +62,6 @@ extends:
                 env:
                   DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
                   DOTNET_CLI_TELEMETRY_OPTOUT: 1
-                  DOTNET_MULTILEVEL_LOOKUP: 0
\ No newline at end of file
+                  DOTNET_MULTILEVEL_LOOKUP: 0
+              - pwsh: ./eng/packages/http-client-csharp/eng/scripts/Test-Packages.ps1
```

Comment:
Give it's just "dotnet test", I think it's not worth to have a script for it.
Taken L48-L51 for example, we can just put "dotnet test" and specify the workingDirectory.